lib/data/models/asset_type.dart

```diff
@@ -1,5 +1,11 @@
 import 'package:freezed_annotation/freezed_annotation.dart';
 
+/// This map needed because all input-related data stores in auto-generated
+/// webview models
+const assetTypeMap = {
+  'tip3_token': AssetType.tip3Token,
```

Comment:
We already have 'tip3_token' as string in the AssetType. Should we use one const string for the both cases?

Reply:
-